src/feature/featurelist.h

```diff
@@ -214,18 +225,37 @@ FEATURE(unsecuredNetworkNotification,      // Feature ID
         FeatureCallback_true,              // Can be flipped on
         FeatureCallback_false,             // Can be flipped off
         QStringList(),                     // feature dependencies
-        FeatureCallback_unsecuredNetworkNotification)
+        enableForPlatform(
+            {.windows = true, .macos = true, .linux = true, .wasm = true}))
 
 FEATURE(webPurchase,           // Feature ID
         "Web Purchase",        // Feature name
         FeatureCallback_true,  // Can be flipped on
         FeatureCallback_true,  // Can be flipped off
         QStringList(),         // feature dependencies
-        FeatureCallback_webPurchase)
-
-FEATURE(localProxy,                      // Feature ID
-        "LocalProxy",                    // Feature name
-        FeatureCallback_proxyCanTurnOn,  // Can be flipped on
-        FeatureCallback_true,            // Can be flipped off
-        QStringList("splitTunnel"),      // feature dependencies
+        enableForPlatform({
+            .windows = true,
+            .macos = true,
+            .linux = true,
+        }))
+
+FEATURE(localProxy,    // Feature ID
+        "LocalProxy",  // Feature name
+        enableForPlatform({
+            .windows = true,
+            .linux = true,
+        }),                          // Can be flipped on
+        FeatureCallback_true,        // Can be flipped off
+        QStringList("splitTunnel"),  // feature dependencies
         FeatureCallback_inStaging)
+
+FEATURE(webExtension,      // Feature ID
+        "webExtension",    // Feature name
+        enableForPlatform({// Can be flipped on
+                           .windows = true,
+                           .macos = true,
+                           .linux = true}),
+        FeatureCallback_true,  // Can be flipped off
+        QStringList(),         // feature dependencies
+        enableForPlatform({    // default value
```

Comment:
This means "default true for windows, default false elsewhere", right? The naming `enableForPlatform` starts to feel a little weird - perhaps `platformBoolean` or something like that?